.github/workflows/ci.yml

```diff
@@ -19,3 +19,50 @@ jobs:
 
     - name: Compile and Test
       run: ./gradlew build
+
+    - name: Upload JAR
+      uses: actions/upload-artifact@v3
+      with:
+          name: rsyncit-jar
+          path: build/libs/rsyncit-0.0.1-SNAPSHOT.jar
+
+
+  docker:
+    runs-on: ubuntu-latest
+    needs: gradle-build
+#    if: ${{ github.ref == 'refs/heads/main' }}
+
+    steps:
+      - name: checkout sources
+        uses: actions/checkout@v3
+
+      - name: Fetch jar
+        uses: actions/download-artifact@v3
+        with:
+          name: rsyncit-jar
+
+      - name: Login to GHCR
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set up Docker Build
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build docker image and push
+        id: docker_build_rsyncit
+        uses: docker/build-push-action@v3
+        with:
+          context: ./
+          file: ./Dockerfile
+          push: true
+          tags: ghcr.io/ripe-ncc/rsyncit:latest
```

Comment:
For `rpki-publication-server` we push an `unstable` tag on every build of `main` and then use `latest` to point to the latest release. Would be good to be consistent with our image tags :)

Reply:
Yeap, done.